README.md

```diff
@@ -17,14 +17,84 @@ An incomplete list of other repositories follows below:
 * https://github.com/openhab/org.openhab.binding.zigbee
 * https://github.com/openhab/openhab-webui
 
-## Build
+## Development / Repository Organisation
 
 openHAB 2 add-ons are [Java](https://en.wikipedia.org/wiki/Java_(programming_language)) `.jar` files.
 
 The openHAB 2 build system is based on [maven](https://maven.apache.org/what-is-maven.html).
 The official IDE (Integrated development environment) is Eclipse.
 
+You find the following repository structure:
+
+```
+.
++-- addons/  Legacy addons that are currently ported to the new buildsystem
+|
++-- bom      Maven buildsystem: Bill of materials
+|   +-- openhab-addons  Lists all extensions for other repos to reference them
+|   +-- ...             Other boms
+|
++-- bundles  Official openHAB extensions
+|   +-- org.openhab.binding.airquality
+|   +-- org.openhab.binding.astro
+|   +-- ...
+|
++-- features/karaf  An extension usually has dependencies (at least openHAB core).
+|            |      In those feature files are the dependencies for the OSGi container declared.
+|            +-- openhab-addons-external/src/main/feature/feature.xml
+|            +-- openhab-addons/src/main/feature/feature.xml
+|
++-- itests   Integration tests. Those tests require parts of the framework to run.
+|   +-- org.openhab.binding.astro.tests
+|   +-- org.openhab.binding.avmfritz.tests
+|   +-- ...
+|
++-- poms     Maven buildsystem files
++-- src/etc  Auxilary buildsystem files: The license header for automatic checks for example
++-- tools    Static code analyser instructions
+|
++-- CODEOWNERS  This file assigns people to directories so that they are informed if a pull-request
+                would modify that directory/binding.
+```
+
+### Command line build
+
 To build all add-ons from the command-line, type in:
-`mvn clean install`
+`mvn clean install -DskipChecks -DskipTests`. This will only build all add-ons but does not run the tests and the static code checks. For binding development you want to run that command without skipping checks and tests.
```

Comment:
I would not want to promote skip tests and checks so actively...

Reply:
If you just want to build jar files (remember that this repo is existing as forks on other developers git repositories) skipping tests and checks is reasonable I think (7 minutes vs 30 minutes). But yeah can be more emphasized that it is not a good idea to develop yourself without those helpers. I'm happy for suggestions.

Reply:
To build all add-ons from the command-line, type in:

`mvn clean install`

Optionally you can skip tests (`-DskipTests`) or skip some static analysis (`-DskipChecks`) this does  improve the build time but could hide problems in your code. For binding development you want to run that command without skipping checks and tests.